res/style.css

```diff
@@ -36,13 +36,19 @@ body, #root, .profileViewer {
   flex: 1;
 }
 
+.treeViewContextMenu {
+  flex: 1;
+  display: flex;
+}
+
 .treeViewBody {
   flex: 1;
   overflow: auto;
   position: relative;
   outline: 0;
   will-change: scroll-position;
   line-height: 16px;
+  z-index: 0;
```

Comment:
I needed to create a new stacking context so that the context menu isn't overlapped by the contents of treeViewBody.